File: backend/src/main/java/com/vira/dply/dto/EnvironmentDto.java
package com.vira.dply.dto;

import java.util.List;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EnvironmentDto {
    private UUID id;
    private String name;
    private String kubeContext;
    private String kubeConfigPath;
    private List<TeamDto> teams;
}

File: backend/src/main/kotlin/com/vira/dply/dto/EnvironmentDto.kt
package com.vira.dply.dto

import java.time.Instant

data class EnvironmentDto(
    val id: Long,
    val name: String,
    var type: String,
    var status: String,
    var kubeconfigRef: String?,
    val createdAt: Instant,
    val updatedAt: Instant
)